projects/fsastorefrontlib/src/core/assets-table-config/assets-table.config.ts

```diff
@@ -0,0 +1,58 @@
+import { AssetTableType } from '../../occ/occ-models/occ.models';
+
+export const getDataByAssetType = (
+  headings
+): { [key in AssetTableType]: any } => ({
+  claims: {
+    headings,
+    values: [
+      { propName: true, value: 'claimNumber' },
+      { propName: true, value: 'insurancePolicy.categoryData.name' },
+      { propName: true, value: 'insurancePolicy.paymentFrequency' },
+      { propName: true, value: 'claimStatus' },
+    ],
+  },
+  policies: {
+    headings: [...headings, 'claim.claim'],
+    values: [
+      { propName: true, value: 'contractNumber' },
+      { propName: true, value: 'categoryData.name' },
+      { propName: true, value: 'paymentFrequency' },
+      { propName: true, value: 'policyStatus' },
+      {
+        propName: false,
+        value: 'CREATE',
```

Comment:
This cannot be fixed like this. When changing language, this will stay the same. Also, shouldn't be in all caps. That should be handled with CSS.